packages/cli/src/daemon-config.ts

```diff
@@ -155,6 +155,22 @@ export class DaemonAnchorConfig {
   ethereumRpcUrl?: string
 }
 
+@jsonObject
+@toJson
+export class DaemonCeramicResolversConfig {
+  /**
+   * Configuration for nft-did-resolver. Its README contains appropriate documentation.
```

Comment:
please add line to the doc comments for all the new fields you added that explain that the name of the field is different in the config file than it is in code.  Something like "When specifying in a config file, use the name 'nft-did-resolver'".

This is the one annoying downside of using the typedjson library to do the field name renames automatically - it makes the doccomments more confusing.

Reply:
ok